### PR TITLE
Feature/2480 add chevron color to drawer

### LINKED ISSUE #327: Chevron and non text right component default color
Describe the bug
Colour for non text right component and chevron is different than **theme.palette.text.secondary** from MUI theme.

What is the expected behaviour?
Default non-text right component and chevron colour should be **theme.palette.text.secondary** from MUI theme.

What are the steps to reproduce?
Use rightComponent props  for InfoListItem in component's storybook.
Check color CSS property

Screenshots:
Screenshot 2021-10-19 at 2 19 32 PM

### PR DESCRIPTION
**Pull request by bkarambe**
Fixes #327 (Same for Drawer)

#### Changes proposed in this Pull Request:
- Added chevronColor prop to Drawer
- Added chevron class to DrawerNavItem

#### Screenshots:
Changed color using prop
Screenshot 2022-01-17 at 4 38 45 PM

#### To Test:
- yarn start:storybook
- Navigate to Drawer and open full config story
- Enable chevron and change chevronColor

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,8 +4,10 @@
 
 ### Added
 
+-   Added `chevronColor` property onto `<Drawer>`.
+-   Added `chevron` class onto `<DrawerNavItem>`.
 -   Added `chevronColor` property onto `<InfoListItem>`.
--   Added `chevronIcon` class for `chevron` element in `<InfoListItem>`.
+-   Added `chevron` class onto `<InfoListItem>`.
```
Comment (joebochill) [inline CHANGELOG.md:10]: Suggestion:

```
-   Added `chevronColor` property to `<InfoListItem>` and SharedProps of `<Drawer>`.
-   Added class override for `chevron` on `<InfoListItem>`
```

File: components/src/core/Drawer/DrawerNavItem.tsx [hunk 2]
```diff
@@ -89,6 +89,11 @@ export type NestedNavItem = NestedDrawerNavItemProps;
 const calcNestedPadding = (theme: Theme, depth: number): number =>
     theme.spacing(depth ? (depth - 1) * 4 : 0) + theme.spacing(2);
 
+const getChevronColor = (props: DrawerNavItemProps, theme: Theme): string => {
+    const { chevronColor } = props;
+    return chevronColor ? chevronColor : theme.palette.text.secondary;
+};
+
```
Comment (joebochill) [inline components/src/core/Drawer/DrawerNavItem.tsx:96]: I see now why we had to do this. I thought we had an issue logged for not being able to combine chevron and rightComponent on the InfoListItem, but it looks like not, so I'll add a new issue for that. What you have here looks fine until we refactor the ILI.

**Comment by jeffvg** [comment]
issue logged here https://github.com/brightlayer-ui/react-component-library/issues/364

**Review by daileytj — APPROVED** [review]
looks good!

**Pull request merged**